### PR TITLE
Us002 view job specification

### PR DESCRIPTION
**Pull request by kamil-biegaj**

### CODE REVIEW

File: service/JobRoleService.ts [hunk 1]
```diff
@@ -7,4 +7,19 @@ module.exports.getJobRoles = async function() {
     } catch (e) {
         return new Error('Could not get job roles.')
     }
-}
\ No newline at end of file
+}
+
+module.exports.getSpecificationById = async function (id: number) { 
+    try {
+        const URL = process.env.UI_URL || 'http://localhost:8080';
```
Comment (shaunganley) [inline service/JobRoleService.ts]: the base url for axios should already be set up to use the UI_URL, you should be able to remove this line

File: service/JobRoleService.ts [hunk 2]
```diff
@@ -7,4 +7,19 @@ module.exports.getJobRoles = async function() {
...
+        const response = await axios.get(URL + '/api/job_roles/' + id)
+        
+        return response.data;
+    } catch (e) {
+        console.log(URL + '/api/job_roles/' + id);
```
Comment (shaunganley) [inline service/JobRoleService.ts]: remove unnecessary logging

File: test/unit/controller/jobRoleControllerTest.ts [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+const expect = chai.expect;
+import sinon from 'sinon';
+
+import { app } from '../../../app';
+const jobRoleService = require('../../../service/JobRoleService');
+
+const mockJobRole = {
+  id: 1,
+  name: 'Software Engineer',
+  specification: 'Intro$Summary$Link',
```
Comment (shaunganley) [inline test/unit/controller/jobRoleControllerTest.ts]: link should be separate property now

File: test/unit/service/jobRoleService.ts [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+   
+    id: "1",
+    name: "Software Engineer",
+    specification: "Does coding.",
+    urlLink: "https://www.google.com"
+    
+}
+
+
+
```
Comment (shaunganley) [inline test/unit/service/jobRoleService.ts]:
```suggestion

```

File: views/view-jobrole-specification.html [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+    <header class="bg-dark text-light">
+        <div class="container">
+            <div class="row">
+                <div class="col-md-12">
+                    <h1>Specification</h1>
+                </div>
+            </div>
+        </div>
+    </header>
+    <main class="container mt-4">
```
Comment (shaunganley) [inline views/view-jobrole-specification.html:20]: add the errormessage section to this view

**Review by shaunganley — CHANGES_REQUESTED** [review]
few small comments

**Comment by kamil-biegaj** [comment]
@ccurran01

**Review by AlexMcC02 — APPROVED** [review]
I see no issues, this looks ready to be merged.

**Review by ccurran01 — APPROVED** [review]

**Pull request merged**